### ISSUE TITLE
Error when importing 0.3.0

### ISSUE THREAD

**Issue by scarlettekk** [google-ai-edge/ai-edge-torch#218]
### Description of the bug:

I was trying to use the latest commit version to convert gemma2 since it seems like 0.2.0 doesn't support it. However, I can't even import it:
```python3
Traceback (most recent call last):
  File "/home/scarlett/git/gemma/convert.py", line 21, in <module>
    import ai_edge_torch
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/__init__.py", line 16, in <module>
    from ai_edge_torch._convert.converter import convert
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/_convert/converter.py", line 21, in <module>
    from ai_edge_torch._convert import conversion
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/_convert/conversion.py", line 20, in <module>
    from ai_edge_torch import lowertools
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/lowertools/__init__.py", line 16, in <module>
    from ._shim import *
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/lowertools/_shim.py", line 25, in <module>
    from ai_edge_torch.lowertools import torch_xla_utils as utils
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/lowertools/torch_xla_utils.py", line 29, in <module>
    from ai_edge_torch.lowertools import translate_recipe
  File "/home/scarlett/.local/share/virtualenvs/gemma-U5bxnpzZ/lib/python3.11/site-packages/ai_edge_torch/lowertools/translate_recipe.py", line 20, in <module>
    _ComputePrecision = quantizer.qtyping.ComputePrecision
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'ai_edge_quantizer.qtyping' has no attribute 'ComputePrecision'
```

### Actual vs expected behavior:

Expected: 
The library imports normally

Actual:
The library fails to import

### Any other information you'd like to share?

Trying to use https://github.com/google-ai-edge/ai-edge-torch/blob/main/ai_edge_torch/generative/examples/gemma/convert_gemma2_to_tflite.py

**Comment by pkgoogle** [comment]
Hi @scarlettekk, how did you install the latest commit?

I was able to import properly the latest commit by doing this:

```sh
cd ai-edge-torch # my github root
git switch main # ensure you are on main branch
git pull # ensure you are up to date
pip install -e . # install the current code in your current environment, python=3.11 was previously installed
python
>>> import ai_edge_torch
```

Does this work if you start in a fresh environment? Thanks.

**Comment by patrickzan** [comment]
Hi @scarlettekk @pkgoogle, I also got the same error with python3.10 environment using the following approach, but when I switched to python3.11 and re-installed ai-edge-torch the same way, the error disappeared. Hope this is helpful. 
```bash
cd ai-edge-torch # my github root
git switch main # ensure you are on main branch
git pull # ensure you are up to date
pip install -e . # install the current code in your current environment, python=3.11 was previously installed
python
>>> import ai_edge_torch
```

**Comment by johndpope** [comment]
what version of tensorflow is necessary? nightly 2.19?

**Comment by johndpope** [comment]
fix here - https://github.com/google-ai-edge/ai-edge-torch/issues/286

**Comment by robertbcalhoun** [comment]
I encountered the same error after running `pip install ai-edge-torch-nightly torchvision` per [Pytorch to TFLite](https://ai.google.dev/edge/litert/models/pytorch_to_tflite) documentation. Fix was #286 as noted by @johndpope:

**Pre-existing state:**
```
$ python -m pip freeze
ai-edge-litert-nightly==1.0.1.dev20250114
ai-edge-quantizer-nightly==0.0.1.dev20240718
ai-edge-torch==0.2.1
ai-edge-torch-nightly==0.3.0.dev20250117
(...)
```

**Fix:**
```
python -m pip uninstall ai-edge-quantizer-nightly==0.0.1.dev20240718
python -m pip install git+https://github.com/google-ai-edge/ai-edge-quantizer
```

**Post:**
```
(env) rcalhoun@topaz:~/convert-pytorch-tflite$ python -m pip freeze
ai-edge-litert-nightly==1.0.1.dev20250114
ai-edge-quantizer @ git+https://github.com/google-ai-edge/ai-edge-quantizer@a3318da7831ba4fb5fda4d714e887c031cb2cac1
ai-edge-torch==0.2.1
ai-edge-torch-nightly==0.3.0.dev20250117
(...)
```
